draft-ietf-quic-transport.md

```diff
@@ -727,16 +732,16 @@ Connection ID.
 
 ### Tokens
 
-If the client has a token received in a NEW_TOKEN frame on a previous connection
-to what it believes to be the same server, it can include that value in the
-Token field of its Initial packet.
+If the client has a token received in a NEW_TOKEN frame on a previous
+connection to what it believes to be the same server, it can include that value
+in the Token field of its Initial packet.
 
-A client SHOULD NOT reuse a token.  Reusing a token on different network paths
-would allow activity to be linked between paths (see {{migration-linkability}}).
-A client MUST NOT reuse a token if it believes that its point of network
-attachment has changed; that is, if there is a change in its local IP address or
-network interface.  A client needs to start the connection process over if it
-migrates prior to completing the handshake.
+A client SHOULD NOT reuse a token.  Reusing a token could allow different
+connections to be identified as coming from the same endpoint (see also
+{{migration-linkability}}).  A client MUST NOT reuse a token if it believes that
+its point of network attachment has changed; that is, if there is a change in
+its local IP address or network interface.  A client MUST start the connection
```

Comment:
I know you didn't change this text, but this seems to preclude clients keeping a cache of tokens and the associated network connections.  For example, a mobile device that went to Wi-Fi could keep the cellular token in expectation that it will be on cellular again.  Perhaps the better prohibition is that you MUST NOT use the token on a different network than where you received it?